include/deal.II/fe/fe_q_bubbles.h

```diff
@@ -0,0 +1,194 @@
+// ---------------------------------------------------------------------
+// $Id$
+//
+// Copyright (C) 2012 - 2013 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+#ifndef __deal2__fe_q_bubbles_h
+#define __deal2__fe_q_bubbles_h
+
+#include <deal.II/base/config.h>
+#include <deal.II/base/tensor_product_polynomials_bubbles.h>
+#include <deal.II/fe/fe_q_base.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+
+/*!@addtogroup fe */
+/*@{*/
+
+/**
+ * Implementation of a scalar Lagrange finite element @p Qp^+ that yields the
+ * finite element space of continuous, piecewise polynomials of degree @p p in
+ * each coordinate direction plus some bubble enrichment space spanned by
+ * $x_j^{p-1}\prod_{i=0}^{dim-1}(1-x_i)^2$. Therefore the highest polynomial
+ * degree is $p+1$.
+ * This class is realized using tensor product polynomials based on equidistant
```

Comment:
So this element is L2-conforming because the basis functions are not zero in all of {x_j=0}. I think it would be helpful to clearly state this in the class description because there exist elements with bubble enrichment that are continuous (H1-conforming).